kernels/quantized/CMakeLists.txt

```diff
@@ -115,6 +115,9 @@ if(NOT CMAKE_GENERATOR STREQUAL "Xcode"
       target_link_libraries(
         quantized_ops_aot_lib PUBLIC quantized_ops_pybind_lib
       )
+      if (_WIN32)
+        target_link_libraries(quantized_ops_aot_lib PUBLIC quantized_ops_lib)
+      endif()
```

Comment:
This doesn't look right. Why is the dependency different for Windows?

Reply:
I don't actually know... if I don't do this, then on Windows there are linker errors where the AOT lib cannot find the symbols for the actual operator implementations in `quantized_ops_lib`.

However, if I do this across the board, then I run into duplicate registration issues on non-Windows platforms. Could be platform dependent linker behaviour, I suppose?

Reply:
I also tested running a model on windows to make sure that the duplicate registration did not occur on Windows (y)